Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.4.17f1
```

Comment:
Are we okay bumping?

Reply:
i thought this was the version we were on. doesn't matter to me which version is used

Reply:
current master is m_EditorVersion: 2018.4.17f1 so we're good here. no change